src/ssda/ssda_populate.py

```diff
@@ -121,16 +154,16 @@ def validate_options(
         )
 
     # Either a date range or a FITS file must be specified
-    if not (start and end) and not file:
+    if not (start and end):
         raise click.UsageError(
             "You must either specify a date range (with the --start/--end options) or "
-            "a FITS file (with the --file option)."
+            "a FITS file."
```

Comment:
"You must specify a start and end date (with the --start/--end options)."

Reply:
Done